Demo/Demo/Models/SetupTokenRequest.swift

```diff
@@ -17,13 +23,24 @@ struct SetUpTokenRequest {
     }
     
     var body: Data? {
-        let requestBody: [String: Any] = [
-            "customer": [
-                "id": customerID
-            ],
-            "payment_source": [
-                "card": [:]
+        var paymentSourceDict: [String: Any] = [:]
+
+        switch paymentSource {
+        case .card:
+            paymentSourceDict["card"] = [:]
+        case .paypal(let usageType):
+            paymentSourceDict["paypal"] = [
+                "usage_type": usageType,
+                "experience_context": [
+                    "return_url": "sdk.ios.paypal://vault/success",
+                    "cancel_url": "sdk.ios.paypal://vault/cancel"
+                ]
             ]
+        }
+
+        let requestBody: [String: Any] = [
+            "customer": ["id": customerID],
+            "payment_source": paymentSourceDict
         ]
 
         return try? JSONSerialization.data(withJSONObject: requestBody)
```

Comment:
Instead of using a raw dict + [JSONSerizliation](https://developer.apple.com/documentation/foundation/jsonserialization), throughout the rest of the demo app we conform structs to `Encodable` & then use [JSONEncoder](https://developer.apple.com/documentation/foundation/jsonencoder)

We should keep things consistent

Reply:
Hey Sammy, I did this because I didn't know a way of using struct to make blank payment JSON object
{
"payment_source":
    "card:" {}
}

Reply:
I know when we get the update setup token endpoint to work with custom url scheme, we will use experience context field but right now, it errors out with hat field so I am sending an empty card in the card vault request.

Reply:
I remember looking at this a while back and not finding a quick answer on `Encodable` support for empty objects. Maybe we can use an empty `struct` to represent `[:]` like [here](https://stackoverflow.com/a/69348020)?

Reply:
I've never seen this. Thanks Steven. I will check it out.

Reply:
We found EmptyParams()